addons/web_editor/controllers/main.py

```diff
@@ -265,7 +265,8 @@ def update_stars(self, res_model, res_id, filename, starsId, rating):
                 star.set('class', classname)
             star_index += 1
 
-        value = etree.tostring(htmlelem[0][0], encoding='utf-8', method='html')[5:-6]
+        value_bytes = etree.tostring(htmlelem[0][0], encoding='utf-8', method='html')[5:-6]
+        value = value_bytes.decode('utf-8')
```

Comment:
Wouldn't the `[5:-6]` need to be done on the string rather that on the bytestring then?

Reply:
I've checked both, and they give the same result. So, we can go with either one.